monday/client.py

```diff
@@ -7,23 +7,34 @@
 """
 
 from .__version__ import __version__
-from .resources import ItemResource, UpdateResource, TagResource, BoardResource, UserResource, GroupResource, ComplexityResource, WorkspaceResource, NotificationResource
+from .resources import CustomResource, ItemResource, ColumnsResource, UpdateResource, TagResource, BoardResource, \
+    UserResource, GroupResource, ComplexityResource, WorkspaceResource, NotificationResource, MeResource
+
+_DEFAULT_HEADERS = {}
 
 
 class MondayClient:
-    def __init__(self, token):
```

Comment:
Curious about this change - is this a preference thing? Monday does call them "tokens" rather than API keys, so token does reflect their internal language.

Reply:
Darnit - we've been using both terms somewhat interchangeably in the code, and I thought I was helping to consolidate us.  However, I went in the wrong direction.  I'm going to make an update to fix that.